.github/workflows/vite.yaml

```diff
@@ -0,0 +1,41 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: client build and test
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  admin-build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
Are you sure it should be `18.x'?

Reply:
18.x versions is LTS, looks like it more preferred
Although potentially this app can be started at ancient with a lot legacy 
In that case i can add additional check, i.e. add node ```14.x``` and ```16.x``` and that will be enough

Reply:
Looks like client packages too new for node 14
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/101043571/235324736-0b99e022-11e4-40de-b6fa-d5433858f0d5.png">

Reply:
@Andrew-Sem I seem to have confused you, I'm talking about `x` in the version.

I asked if you could put `x` instead of a specific number?

Reply:
@l3r8yJ in this case, everything is much simpler. I will change version to exactly LTS